src/utilities/richTextField.js

```diff
@@ -12,6 +12,7 @@ function createMarkup(storyblokHTML) {
     allowedTags: false, // This allows extra HTML tags to be rendered, e.g., H2, img
     allowedAttributes: false, // This allows custom class names
     allowVulnerableTags: true, // https://github.com/apostrophecms/sanitize-html/blob/main/CHANGELOG.md#1250
+    exclusiveFilter: (frame) => frame.tag === 'a' && !frame.text.trim(), // Remove empty links
```

Comment:
Removes empty links. The `santizeHtml` docs use removing empty links as their `exclusiveFilter` example: https://www.npmjs.com/package/sanitize-html#filters 😆

Reply:
ah, that's niiice.

Reply:
@sherakama Unlike [the related Alumni site fix](https://github.com/SU-SWS/saa_alumni/pull/819), this will also remove links that have child elements (e.g. `<br>`) as long as there's no eventual text ancestors. That could cause some unexpected layout shifting.

I think I'm going to close this PR, manually handle the handful of WYSIWYG issues to get rid of the a11y issue, then handle this in the site rebuild instead...